apps/arweave/src/ar_mining_worker.erl

```diff
@@ -199,24 +232,23 @@ handle_cast(handle_task, #state{ task_queue = Q } = State) ->
 			end
 	end;
 
-handle_cast({remove_sub_chunks_from_cache, SubChunkCount, Candidate}, State) ->
-	{noreply, remove_sub_chunks_from_cache(Candidate, SubChunkCount, State)};
+handle_cast(?MSG_REMOVE_SUB_CHUNKS_FROM_CACHE(SubchunkCount, Candidate), State) ->
+	{noreply, remove_subchunks_from_cache(Candidate, SubchunkCount, State)};
```

Comment:
```suggestion
	{noreply, remove_sub_chunks_from_cache(Candidate, SubChunkCount, State)};
```

Reply:
I think we could go either way with `SubChunk` vs `Subchunk`, but in the rest of the codebase we treat it as two words, so best to keep it hat way for now